src/app/js/formats/hierarchy/Hierarchy.js

```diff
@@ -685,6 +698,16 @@ class Hierarchy extends PureComponent {
         this.update();
     }
 
+    handleMouseEnter = () => {
+        this.isHovered = true;
+        alert('enter');
```

Comment:
I hope you plan to finish that part.

Reply:
Dead code written before finding `MenuItem.js` in the app.